helpers/index.js

```diff
@@ -0,0 +1,3 @@
+export const joinArrayAsString = (arr) => {
```

Comment:
Why helper? We already have this method in Array prototype. Please remove

Reply:
Ok